src/tasks/AutoBidAuctionTask.py

```
@@ -68,11 +70,11 @@ def do_run(self):
 
         # 拍卖主界面 UI 元素坐标（多个阶段共享）
         # 开始匹配按钮
-        box_match = self.box_of_screen(0.7427, 0.8972, to_x=0.8360, to_y=0.9472)
+        box_match = self.box_of_screen(0.7427, 0.8972, 0.8360, 0.9472)
         # 确认按钮（匹配成功后的确认）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)